FirelandsProject/cataclysm
# Core/Misc: Update locales_creature_text to simple system

### Changes proposed

- Normalize the `creature_text` and `creature_text_locale` table to adapt them to the new structures of the current emulators.

### Issues addressed


- Closes

### SOURCE:



### Tests performed


- Windows 10
- In-game

### How to Test the Changes:


1. Create a text inside `creature_text_locale` corresponding to the game language of your client, and you should see this text in the set site.

### Known issues and TODO list



---

- Thank you for collaborating with the project.
- For an efficient working methodology.
- It is obvious to make long pull request, unless it is necessary.
- A short pull request would be easy to test and approve.
